back-end/Controllers/UsersController.cs

```diff
@@ -120,6 +120,101 @@ public async Task<IActionResult> RemoveStock([FromBody] string id, string symbol
         return NoContent();
     }
 
+    [HttpPut("sendRequest/{receiverEmail}")]
+    public async Task<IActionResult> SendRequest([FromBody] string id, string receiverEmail)
+    {
+        User? sendingUser = await _usersService.GetAsyncById(id);
+        User? receivingUser = await _usersService.GetAsync(receiverEmail);
+
+        if (sendingUser is null || receivingUser is null) return NotFound();
+
```

Comment:
@OlivierPoitras You have to replace all NotFound for NoContent() otherwise it will show a 404 error in the user log. We spoked about this

Reply:
@OlivierPoitras j’ai fermé les autres comments, il reste juste quelques NotFound() à remplacer dans ce fichier and I think that’s good!

Reply:
Nice work! 📈